packages/expo-sqlite/ios/ExpoSQLite.podspec

```
@@ -15,7 +15,7 @@ Pod::Spec.new do |s|
   s.static_framework = true
   s.dependency 'ExpoModulesCore'
   # The builtin sqlite does not support extensions so we update it
-  s.dependency 'sqlite3', '3.39.2'
+  s.dependency 'sqlite3', '3.42.0'
```

What about this 👇? This can reduce the number of cases where the versions are conflicting with each other (especially if we decide to bump its patch version)
```suggestion
  s.dependency 'sqlite3', '~> 3.42.0'
```